src/backend/Dockerfile

```diff
@@ -22,19 +22,43 @@ RUN set -ex \
         "libgdal-dev" \
     && rm -rf /var/lib/apt/lists/*
 
-# Install PDM
-RUN pip install -U pdm
-# Disable update check
-ENV PDM_CHECK_UPDATE=false
-
 WORKDIR /project
 
+# Install uv
+COPY --from=ghcr.io/astral-sh/uv:0.5.2 /uv /bin/uv
```

Comment:
Does this definitely work? I'm sure you tested it and I just didn't realise this syntax was possible 😅 

But I thought that `--from` and external source never used to be an option!

That's why I used this pattern:
```dockerfile
FROM ghcr.io/astral-sh/uv:0.5.2 as uv-image
...
COPY --from=uv-image /uv /bin/uv
```

Reply:
Nice! Thus must have been added in a recent docker version 
https://docs.docker.com/build/building/multi-stage/#use-an-external-image-as-a-stage